lib/vagrant-lxc/plugin.rb

```diff
@@ -8,12 +8,16 @@ class Plugin < Vagrant.plugin("2")
       The LXC provider allows Vagrant to manage and control
       LXC-based virtual machines.
       EOF
+      locale_loaded = false
 
       provider(:lxc, parallel: true, priority: 7) do
```

Comment:
Yeah, that sounds good to me. The docker provider is the newest in the main vagrant project and generally a good example to follow.